common/types/db-types.ts

```diff
@@ -2,6 +2,10 @@ type Id = {
   readonly id: string;
 };
 
+type Type = {
```

Comment:
Are we thinking of adding more inner types to this parent `Type`? Not sure why it's a single type nested inside another. Also, `Type` is a very vague name; maybe consider something more specific